components/Breadcrumbs.js

```diff
@@ -0,0 +1,8 @@
+import {Breadcrumb} from 'react-bootstrap';
```

Comment:
Would be nice if we renamed the file to `Breadcrumb.js` (without the `s`) so that it matches the component name.